pkg/helpers/conditions/conditions.go

```diff
@@ -27,6 +28,12 @@ func PodContainerRunning(containerName string) watchtools.ConditionFunc {
 		case *corev1.Pod:
 			switch t.Status.Phase {
 			case corev1.PodRunning, corev1.PodPending:
+				if t.Status.ContainerStatuses[0].State.Waiting != nil {
+					if t.Status.ContainerStatuses[0].State.Waiting.Reason == "CreateContainerError" {
+						klog.V(0).Info(t.Status.ContainerStatuses[0].State.Waiting.Message)
```

Comment:
More info what how and why, same applies for the next error.

Reply:
I've added info